workloads/jenkins/Jenkinsfile-multi-nightly

```diff
@@ -56,11 +56,6 @@ pipeline {
              bash -x ./workloads/jenkins/scripts/test-local-registry.sh
           '''
 
-          echo "Test Rook+Ceph installation"
-          sh '''
-             timeout 2200 bash -x ./workloads/jenkins/scripts/test-ceph.sh
```

Comment:
I made the decision to only test the rook-ceph install in the single-mgmt node nightly to save on testing resources/complexity.